agent/stats/utils_unix.go

```diff
@@ -43,7 +43,11 @@ func dockerStatsToContainerStats(dockerStats *types.StatsJSON) (*ContainerStats,
 	}, nil
 }
 
+// Safe return of storage stats based on Docker Stats (Unix)
 func getStorageStats(dockerStats *types.StatsJSON) (uint64, uint64) {
+	if dockerStats.BlkioStats.IoServiceBytesRecursive == nil {
```

Comment:
I believe we should also do the following check: 
`if dockerStats.BlkioStats == nil`

Reply:
I'll add that.